pkg/services/provisioning/dashboards/file_reader_linux_test.go

```diff
@@ -25,7 +25,9 @@ func TestProvisionedSymlinkedFolder(t *testing.T) {
 	}
 
 	reader, err := NewDashboardFileReader(cfg, log.New("test-logger"), nil)
-	require.NoError(t, err)
+	if err != nil {
+		t.Error("expected err to be nil")
+	}
```

Comment:
I think the existing code makes more sense. Any reason you changed it?

```suggestion
	require.NoError(t, err)
```

Reply:
Will merge as is for now